pkgs/development/libraries/v8/6_x.nix

```diff
@@ -1,4 +1,4 @@
-{ stdenv, lib, fetchgit, fetchFromGitHub, gn, ninja, python, glib, pkgconfig
+{ stdenv, lib, cctools, fetchgit, fetchFromGitHub, gn, ninja, python, glib, pkgconfig
```

Comment:
Curious what you need `cctools` for here, since most of its tools should be in scope already from the `stdenv`. Not saying you don't but it surprises me!

Reply:
it contains `libtool` which interestingly isn't in scope otherwise

Reply:
I overhear people talk about this Darwin vs GNU libtool a lot (or is it just a really old version?). Maybe there is an attributes already?

Reply:
so after you've mentioned it I've tried again with GNU libtool, this is how the build fails then:

```
[27/1119] LIBTOOL-STATIC obj/build/config/sanitizers/liboptions_sources.a
FAILED: obj/build/config/sanitizers/liboptions_sources.a 
rm -f obj/build/config/sanitizers/liboptions_sources.a && TOOL_VERSION=1508177652 python ../../build/toolchain/mac/filter_libtool.py libtool -static  -o obj/build/config/sanitizers/liboptions_sources.a obj/build/config/sanitizers/options_sources/sanitizer_options.o
Usage: /nix/store/y9f4wn79hrhdly3yb5766byrd7h398yy-libtool-2.4.6/bin/libtool [OPTION]... [MODE-ARG]...
Try 'libtool --help' for more information.
libtool:   error: unrecognised option: '-static'
```

IMHO this is because V8's build system assumes it's using darwin's libtool which behaves differently. As it probably doesn't make sense to patch V8's build system to use GNU's libtool (actually I don't now if that's feasible at all) I'd vote to just stick with darwin's version!?

Reply:
Oh sure, I meant maybe there was like a `darwin.libtool` or something, not that you should shoehorn in GNU's libtool.

Reply:
I've changed it to `libtool`in the argument list and do the aliasing in `all-packages.nix` file